VERSIONING.md

```diff
@@ -73,7 +73,7 @@ respect to semantic versioning.
 |----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Java     | 8+                                                                                                                                                                                                                               | All artifacts, unless otherwise noted                   | Changing requires major version bump.                                                                                                                                              |
 | Android  | 21+ (NOTE: [desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring) is required. We stay up to date with the latest version of [desugar_jdk_libs](https://github.com/google/desugar_jdk_libs).) | Artifacts using `otel.animalsniffer-conventions` plugin | Kept in sync with minimum requirements for [Google Play services](https://developers.google.com/android/guides/setup). Subject to change in minor version.                         |
-| Kotlin   | 1.6+                                                                                                                                                                                                                             | Only applies to `opentelemetry-extension-kotlin`        | Kept in sync with [minimum non-deprecated](https://kotlinlang.org/docs/gradle-compiler-options.html#attributes-common-to-jvm-and-js) version. Subject to change in minor versions. |
+| Kotlin   | 1.8+                                                                                                                                                                                                                             | Only applies to `opentelemetry-extension-kotlin`        | Kept in sync with [minimum non-deprecated](https://kotlinlang.org/docs/gradle-compiler-options.html#attributes-common-to-jvm-and-js) version. Subject to change in minor versions. |
```

Comment:
1.8 is the minimum non-deprecated version as of `kotlin-gradle-plugin-api-2.1.10`: https://github.com/JetBrains/kotlin/blob/6dff5659f42b0b90863d10ee503efd5a8ebb1034/libraries/tools/kotlin-gradle-compiler-types/src/generated/kotlin/org/jetbrains/kotlin/gradle/dsl/KotlinVersion.kt#L17

Though on master, 2.0 is the minimum non-deprecated version: https://github.com/JetBrains/kotlin/blob/a00650ccd44933f7bf3421e912cdedb7a5cae974/libraries/tools/kotlin-gradle-compiler-types/src/generated/kotlin/org/jetbrains/kotlin/gradle/dsl/KotlinVersion.kt#L19